docs/install/install-aur.md

```diff
@@ -6,6 +6,42 @@ sidebar_position: 4
 
 # Installing Vignette using AUR
 
-something
+If you wish to install Vignette on your Linux computer, you can use any AUR helper of your choice.
+
+The package is [`vignette-git`](https://aur.archlinux.org/packages/vignette-git/).
+
+Here I'll be using `yay`.
+
+
+
+## Installation process
+
+In order to install Vignette using yay, run the following command:
+
+
+`yay vignette-git`
+
+
+
+If more than one package appears you should find the right package, which should be listed as `aur/vignette-git`. In this example, that package is number `1`.
+
+![](/img/install/yay-terminal-window.webp)
```

Comment:
Add an alt text here just in case we have people who has failure loading the image